stm/stm.ml

```diff
@@ -65,7 +65,7 @@ end
 
 (* During interactive use we cache more states so that Undoing is fast *)
 let interactive () =
-  if !Flags.ide_slave || !Flags.print_emacs || not !Flags.batch_mode then `Yes
+  if !Flags.ide_slave || not !Flags.batch_mode then `Yes
```

Comment:
PG uses a different path to deal with auto-compilation so -emacs -compile is probably useless and we can make it an error.
Unless people put -compile in the _coqProject file? That would raise an error in situations where it only altered the behaviour of coqtop.